src/Tasks/Engines/DoormanRunner.php

```diff
@@ -63,7 +63,7 @@ public function runQueue($queue)
             BASE_PATH . "/vendor/silverstripe/framework/cli-script.php dev/tasks/ProcessJobQueueChildTask"
         );
         $logPath = Environment::getEnv('SS_DOORMAN_LOGPATH');
-        if ($logPath) {
+        if ($logPath && is_dir($logPath)) {
```

Comment:
What happens if the dir doesn't yet exist?  It's not automatically created somewhere via `mkdir()`?

Reply:
No it's not, it's just appended to the stdOut and stdErr calls if it's set.